packages/client/src/error/code.rs

```diff
@@ -0,0 +1,5 @@
+pub const UNKNOWN: i32 = 1000;
```

Comment:
There are three cases where we are constructing errors explicitly:
- converting from `dyn std::error::Error`
- converting from JS expections (in the compiler and runtime)
- converting from serde errors
While the `code` field is optional, these codes represent those four cases.